apps/callscreen/js/handled_call.js

```diff
@@ -142,7 +148,7 @@ function hc_computeCallEndedFontSizeRules() {
     '@media (min-height: 4.5em) {' +
     '  #calls.big-duration > section.ended .duration > span,' +
     '  #calls.big-duration > section.ended .total-duration {' +
-    '    font-size: ' + (info.fontSize / 10.0) + 'rem;' +
+    //'    font-size: ' + (info.fontSize / 10.0) + 'rem;' +
```

Comment:
I commented it out because it was breaking a test for an unknown reason. I'll fix it before asking for review